test/functional/ui/spell_spec.lua

```diff
@@ -140,6 +141,43 @@ describe("'spell'", function()
       {0:~                                                                               }|
                                                                                       |
     ]])
+    -- Folding an empty line does not remove Cap in next line
```

Comment:
The description of this test need to be changed. I think just `SpellCap highlight` is a better name.